utils.py

```diff
@@ -285,31 +302,31 @@ def get_package_manager():
         config.SUPERUSER_COMMAND = "doas"
 
     # Check for package manager and associated packages
-    if shutil.which('apt') is not None: # debian, ubuntu
+    if shutil.which('apt') is not None:  # debian, ubuntu
         config.PACKAGE_MANAGER_COMMAND_INSTALL = "apt install -y"
-        config.PACKAGE_MANAGER_COMMAND_QUERY = "dpkg -l | grep -E '^.i  '" #IDEA: Switch to Python APT library? See https://github.com/FaithLife-Community/LogosLinuxInstaller/pull/33#discussion_r1443623996
+        config.PACKAGE_MANAGER_COMMAND_QUERY = "dpkg -l | grep -E '^.i  '"  # IDEA: Switch to Python APT library? See https://github.com/FaithLife-Community/LogosLinuxInstaller/pull/33#discussion_r1443623996
         config.PACKAGES = "binutils cabextract fuse wget winbind"
-        config.L9PACKAGES= "" # FIXME: Missing Logos 9 Packages
-    elif shutil.which('dnf') is not None: # rhel, fedora
+        config.L9PACKAGES = ""  # FIXME: Missing Logos 9 Packages
+    elif shutil.which('dnf') is not None:  # rhel, fedora
         config.PACKAGE_MANAGER_COMMAND_INSTALL = "dnf install -y"
         config.PACKAGE_MANAGER_COMMAND_QUERY = "dnf list installed | grep -E ^"
         config.PACKAGES = "patch mod_auth_ntlm_winbind samba-winbind samba-winbind-clients cabextract bc libxml2 curl"
-        config.L9PACKAGES = "" # FIXME: Missing Logos 9 Packages
-    elif shutil.which('yum') is not None: # rhel, fedora
+        config.L9PACKAGES = ""  # FIXME: Missing Logos 9 Packages
+    elif shutil.which('yum') is not None:  # rhel, fedora
         config.PACKAGE_MANAGER_COMMAND_INSTALL = "yum install -y"
-        config.PACKAGE_MANAGER_COMMAND_QUERY = "yum list installed | grep -E ^" #TODO: Needs testing. Is anyone still using `yum`?
+        config.PACKAGE_MANAGER_COMMAND_QUERY = "yum list installed | grep -E ^"  # TODO: Needs testing. Is anyone still using `yum`?
         config.PACKAGES = "patch mod_auth_ntlm_winbind samba-winbind cabextract bc libxml2 curl"
-        config.L9PACKAGES = "" # FIXME: Missing Logos 9 Packages
-    elif shutil.which('pamac') is not None: # manjaro
+        config.L9PACKAGES = ""  # FIXME: Missing Logos 9 Packages
+    elif shutil.which('pamac') is not None:  # manjaro
         config.PACKAGE_MANAGER_COMMAND_INSTALL = "pamac install --no-upgrade --no-confirm"
         config.PACKAGE_MANAGER_COMMAND_QUERY = "pamac list -i | grep -E ^"
         config.PACKAGES = "patch wget sed grep gawk cabextract samba bc libxml2 curl"
-        config.L9PACKAGES = "" # FIXME: Missing Logos 9 Packages
-    elif shutil.which('pacman') is not None: # arch, steamOS
+        config.L9PACKAGES = ""  # FIXME: Missing Logos 9 Packages
+    elif shutil.which('pacman') is not None:  # arch, steamOS
         config.PACKAGE_MANAGER_COMMAND_INSTALL = 'pacman -Syu --overwrite \* --noconfirm --needed'
```

Comment:
python3.12 gives a `SyntaxWarning` about `\*` in this line. I making it a raw string; i.e. `r'pacman ...'` . I did this already for the reg file strings that also have backslashes in them.